bokehjs/src/coffee/document.coffee

```diff
@@ -573,7 +573,7 @@ class Document
       throw new Error("JSON object has wrong type #{typeof json}")
     py_version = json['version']
     versions_string = "Library versions: JS (#{js_version})  /  Python (#{py_version})"
-    if js_version.split('-')[0] != py_version.split('-')[0]
+    if js_version != py_version
```

Comment:
unreleased dev versions will have a dash, though, and we need to not-do this test in that case. This would be the case e.g. running off master with inline resources.